src/helpers/configuration_helpers.sh

```diff
@@ -199,8 +209,23 @@ function show_or_update_perennial_branches {
 }
 
 
+# shellcheck disable=SC2120
+function show_or_update_hack_push_strategy {
+  local strategy=$1
+  if [ -z "$strategy" ]; then
+    echo "$HACK_PUSH_STRATEGY"
+  elif [ "$strategy" != 'push' ] && [ "$strategy" != 'local' ]; then
+    echo "Invalid git-hack push strategy: '$strategy'."
+    echo "Valid git-hack push strategies are 'push' and 'local'."
+  else
+    store_configuration hack-push-strategy "$strategy"
+  fi
```

Comment:
Cloned from https://github.com/Originate/git-town/pull/721/files/18788ed25b549b42a452c61e43bd420041367847#diff-027deb499d8395c56e09b9d03d851a3aR229 :grimacing: Did you want that DRYed up? I'm thinking the overhead isn't worth it.